packages/transformers/js/src/JSTransformer.js

```diff
@@ -3,7 +3,7 @@ import type {JSONObject, EnvMap} from '@parcel/types';
 import type {SchemaEntity} from '@parcel/utils';
 import SourceMap from '@parcel/source-map';
 import {Transformer} from '@parcel/plugin';
-import {transform} from './native';
+import {init, transform} from '../native';
```

Comment:
`init` only exists in the wasm version?

Reply:
Yes. It's a promise (not a function returning a promise), so it's not really needed to add that stub export for the non-wasm version.

It's only needed in the web wasm version (Node wasm is synchronous and native of course as well).